src/CircuitZoo/CircuitZoo.jl

```diff
@@ -37,6 +37,13 @@ struct Purify2to1 <: AbstractCircuit
     end
 end
 
+struct Purify3to1 <: AbstractCircuit
+    leaveout::Symbol
+    function Purify3to1()
```

Comment:
I am not quite sure this constructor really works. Something like `struct Purify3to1 <: AbstractCircuit end` would be simpler and should work fine.